clients/MYSQL/mysql2.js

```diff
@@ -0,0 +1,14 @@
+const { filterObject } = require('../../utils/index.js');
+
+const mapping = {
+  host: 'host',
+  port: 'port',
+  username: 'user',
+  database: 'database',
+  password: 'password'
+};
+
+module.exports = {
+  mapping,
+  filter: (binding) => filterObject(binding, Object.values(mapping))
```

Comment:
I don't think this woud do the right filtering in the general case (where the mapping is not a strict one-to-one), but is ok in this specific case.

Reply:
Yes, so the filter function acts as a wrapper where it allows us to set a custom filtering per client